### ISSUE TITLE
TypeError on pcrypt.js:88

### ISSUE THREAD

**Issue by Moosheplusplus** [laverdet/pcrypt#1]
```
pcrypt/pcrypt.js:88
        return Buffer.from(outputBuffer).slice(0, output8.length - output8[output8.length - 1]);
                      ^

TypeError: this is not a typed array.
  at Function.from (native)
  at Object.decrypt (pcrypt/pcrypt.js:88:17)
```

from what I've gathered, this is thrown due to an outdated version of [nodejs (< 6.0.0)](https://nodejs.org/docs/v6.0.0/api/buffer.html#buffer_buffer_from_buffer_alloc_and_buffer_allocunsafe), and can be fixed by instantiating a Buffer instance as `new Buffer(arrayBuffer)`.

**Comment by trisk** [comment]
The entire statement can be replaced with:
`return new Buffer(outputBuffer, 0, output8.length - output8[output8.length - 1]);`
or for node >= 6.0.0
`return Buffer.from(outputBuffer, 0, output8.length - output8[output8.length - 1]);`

**Comment by Moosheplusplus** [comment]
@trisk you're right. done. :+1:

**Comment by laverdet** [comment]
Merged 6f52723f6a0244f9cd6c4c0e339d475bb6ba7855, thanks
